BackBLAST

```diff
@@ -496,12 +550,21 @@ function main() {
     perform_run ${@:2}
   elif [[ ${run_mode} = "auto" ]]; then
     perform_auto ${@:2}
+  elif [[ ${run_mode} = "search" ]]; then
```

Comment:
If the user does call one of the utility functions through `BackBLAST` (see comment above), I just run that utility script. The user then gets the help message from that script.

The help messages for the individual scripts aren't standardized (i.e., they all look a little different). However, it's hard to get them to all look consistent when the scripts are written in different languages.